### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by Sulayman6907**
Have a little bit of issues adjusting the UI will discuss with you in the morning.

### CODE REVIEW

File: src/Components/App.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import Main from './Main'
+
+const App = () => {
+  return (
+    <Container>
+      <Wrapper>
+      <Sidebar/>
+      <Main/>
+      </Wrapper>
+
```
Comment (Lutif) [inline src/Components/App.js:14]: ?

File: src/Components/App.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import styled from 'styled-components'
+import bgImg from '../assests/bg.png'
+import Sidebar from './Sidebar'
+import Main from './Main'
```
Comment (Lutif) [inline src/Components/App.js:5]: why default imports ?

**Review by Lutif — CHANGES_REQUESTED** [review]
I specifically gave you a set of rules. why aren't you following them?  It makes no sense to review the code.
Screenshot 2021-07-12 at 10 38 36 AM

File: src/Components/App.js (hunk 1 above)
Comment (Sulayman6907) [reply src/Components/App.js:14]: This was made and pushed before the guidelines were given I corrected the default exports in the next pull request.

**Pull request closed without merging**